# Request
bool attach_to_shelf
---
# Respons
bool complete